api/src/main/scala/hmda/api/protocol/processing/SubmissionProtocol.scala

```diff
@@ -0,0 +1,54 @@
+package hmda.api.protocol.processing
+
+import hmda.model.fi._
+import hmda.api.model.Submissions
+import spray.json.{ DefaultJsonProtocol, DeserializationException, JsString, JsValue, RootJsonFormat }
+
+trait SubmissionProtocol extends DefaultJsonProtocol {
+
+  implicit object SubmissionStatusJsonFormat extends RootJsonFormat[SubmissionStatus] {
+    override def write(status: SubmissionStatus): JsValue = {
+      status match {
+        case Created => JsString("created")
+        case Uploading => JsString("uploading")
+        case Uploaded => JsString("uploaded")
+        case Parsing => JsString("parsing")
+        case Parsed => JsString("parsed")
+        case ValidatingSyntaxAndValidity => JsString("validating syntactical and validity")
+        case ValidatedSyntaxAndValidity => JsString("validated syntactical and validity")
+        case ValidatingQualityAndMacro => JsString("validating quality and macro")
+        case Unverified => JsString("unverified")
+        case Verified => JsString("verified")
+        case Signed => JsString("signed")
+        case Failed(msg) => JsString(s"failed: $msg")
+
```

Comment:
I was going to suggest replacing all these cases with:
```
        case Failed(msg) => JsString(s"failed: $msg")
        case otherStatus => JsString(otherStatus.message)
```
... but then I noticed the various `SubmissionStatus.message` values don't actually match these strings.  Which in turn got me wondering: what are those messages for, then?  They are currently unused.